linode/manager
# M3-5390: Update Add a Payment drawer to support multiple payment methods of same type

## Description

- Allows users to add multiple Credit Cards or Google Pay cards to their Linode account
- Users can have at most 6 payment methods
- Display a warning and disable Forms and Buttons in the `Add Payment Method` drawer if user has >= 6 payment methods

## How to test

- Try adding payment methods (Google Pay and Credit Cards)
- Try adding more than 6 payment methods
  - You should see a warning and all add buttons and forms should be disabled
- Test on an account with 0 payment methods

## Demo
https://user-images.githubusercontent.com/6440455/129086004-9eee6edc-282d-456f-92e8-67bfb0edd268.mov

#### Add Payment Method Drawer when user has >= 6 payment methods
![Screen Shot 2021-08-11 at 2 54 04 PM](https://user-images.githubusercontent.com/6440455/129086617-3f8d1e21-2bcc-4706-9c5a-2598f3675418.png)
